### ISSUE TITLE
Tooltip doesn't work in IE11

### ISSUE THREAD

**Issue by diogen737** [valor-software/ng2-bootstrap#5553]
**Bug description:**

Basic example of tooltip usage does not work in my app.

```
<button type="button" class="btn btn-primary"
        tooltip="Vivamus sagittis lacus vel augue laoreet rutrum faucibus.">
  Simple demo
</button>
```

Tooltip does not appear when I hover over the button, the tab freezes infinitely and I also observe intensive CPU usage. No error messages in console.

Other ngx-bootstrap components seem to work fine.

**Versions of ngx-bootstrap, Angular, and Bootstrap:**

ngx-bootstrap: 5.1.1

Angular: 8.2.2

Bootstrap: 4

Build system: Angular CLI

**Comment by gokhanoner** [comment]
Similar problem. For me, I'm using the templated tooltip. I can see the tooltip but when moved cursor, tooltip stays open & freezes the browser.

Angular: 7.2.3
ngx-bootstrap >+3.0.0 (I observed the problem in 5.3.3 & tested all version. Currently using 2.0.5 & working fine.)

Bootstrap: 3

**Comment by gokhanoner** [comment]
update: I removed `container="body"` and now tooltip is working with IE11.

**Comment by diogen737** [comment]
Eventually I've managed to figure this one out.
I have IntersectionObserver running on this page (with polyfill for IE, obviously). Turned out that for some reason the tooltip trigger made the page crash UNLESS the observer had fired the intersection event. The funny thing is that this observer is registered for entirely different element and once I disabled it the tooltip started to show normally.

So actually I don't really know whether the problem is with the tooltip or IO polyfill, but at least the problem is solved for my case and I can close the issue.
